ui/apps/platform/src/Containers/ComplianceEnhanced/Dashboard/ScanResultsOverviewTable.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useState } from 'react';
+import {
+    TableComposable,
+    Tbody,
+    Td,
+    Th,
+    Thead,
+    Tr,
+    ActionsColumn,
+    IAction,
```

Comment:
Linda mentioned that lint for cloud service enforces alphabetical order.

Although it is better to rely on lint rules than comments, can you put in order in case ui adopts that rule?

Reply:
Dang I even intended to alphabetize everything but missed that. I'm kind of surprised we don't have a lint rule set up for that. Will update in next commit.

Reply:
Because Visual Studio is so helpful, we do need that rule. We can ask in team meeting this afternoon.